ladybug_geometry/geometry2d/arc.py

```diff
@@ -0,0 +1,400 @@
+
+    @property
+    def is_inverted(self):
+        """Boolean noting whether the end angle a2 is smaller than the start angle a1."""
+        return self._a2 < self._a1
+
+    def move(self, moving_vec):
+        """Get an arc that has been moved along a vector.
+
+        Args:
+            moving_vec: A Vector2D with the direction and distance to move the arc.
+        """
+        return Arc2D(self.c.move(moving_vec), self.r, self.a1, self.a2)
+
+    def rotate(self, angle, origin):
+        """Get a arc that is rotated counterclockwise by a certain angle.
+
+        Args:
+            angle: An angle for rotation in radians.
+            origin: A Point2D for the origin around which the arc will
+                be rotated.
+        """
+        _a1 = self.a1 + angle
+        _a2 = self.a2 + angle
+        _a1 = _a1 - 2 * math.pi if _a1 > 2 * math.pi else _a1
+        _a2 = _a2 - 2 * math.pi if _a2 > 2 * math.pi else _a2
+        return Arc2D(self.c.rotate(angle, origin), self.r, _a1, _a2)
+
+    def reflect(self, normal, origin):
+        """Get a arc reflected across a plane with the input normal vector and origin.
+
+        Args:
+            normal: A Vector2D representing the normal vector for the plane across
+                which the arc will be reflected. THIS VECTOR MUST BE NORMALIZED.
+            origin: A Point2D representing the origin from which to reflect.
+        """
+        return Arc2D.from_start_mid_end(self.p2.reflect(normal, origin),
+                                        self.midpoint.reflect(normal, origin),
+                                        self.p1.reflect(normal, origin))
+
+    def scale(self, factor, origin=None):
+        """Scale a arc by a factor from an origin point.
+
+        Args:
+            factor: A number representing how much the arc should be scaled.
+            origin: A Point2D representing the origin from which to scale.
+                If None, it will be scaled from the World origin (0, 0).
+        """
+        return Arc2D(self.c.scale(factor, origin), self.r * factor, self.a1, self.a2)
+
+    def subdivide(self, distances):
+        """Get Point2D values along the arc that subdivide it based on input distances.
+
+        Args:
+            distances: A list of distances along the arc at which to subdivide it.
+                This can also be a single number that will be repeated unitl the end of
+                the arc.
+        """
+        if isinstance(distances, (float, int)):
+            distances = [distances]
+        arc_length = self.length
+        dist = distances[0]
+        index = 0
+        sub_pts = [self.p1]
+        while dist < arc_length:
+            sub_pts.append(self.point_at_length(dist))
+            if index < len(distances) - 1:
+                index += 1
+            dist += distances[index]
+        sub_pts.append(self.p2)
+        return sub_pts
+
+    def subdivide_evenly(self, number):
```

Comment:
I would suggest to add a `to_polygon` / `to_polyline` method which uses this method under the hood and returns a polygon / polyline created from the list of points.

Reply:
That is a good idea.  Polylines are the next thing that I was planning on adding to the `ladybug_geometry` library so I will add this method to the Arc objects when I add Polylines